src/eterna/mode/PoseEdit/PoseEditMode.ts

```diff
@@ -2152,25 +2159,58 @@ export default class PoseEditMode extends GameMode {
                 libraryNT: this._poses[0].librarySelections ?? []
             }, solToSubmit);
         } else {
-            const NOT_SATISFIED_PROMPT = 'Puzzle constraints are not satisfied.\n\n'
-                + 'You can still submit the sequence, but please note that there is a risk of not getting '
-                + 'synthesized properly';
-
-            if (!this.checkConstraints()) {
-                // If we pass constraints when taking into account soft constraints, just prompt
-                if (this.checkConstraints(this._puzzle.isSoftConstraint || Eterna.DEV_MODE)) {
-                    this.showConfirmDialog(NOT_SATISFIED_PROMPT).closed
-                        .then((confirmed) => {
-                            if (confirmed) {
-                                this.promptForExperimentalPuzzleSubmission();
+            const pipeline = [
+                () => {
+                    if (!this.checkConstraints()) {
+                        // If we pass constraints when taking into account soft constraints, just prompt
+                        if (this.checkConstraints(this._puzzle.isSoftConstraint || Eterna.DEV_MODE)) {
+                            const NOT_SATISFIED_PROMPT = 'Puzzle constraints are not satisfied.\n\n'
+                            + 'You can still submit the sequence, but please note that there is a risk of not getting '
```

Comment:
Suggestion: 
"...a risk of not getting synthesized properly." -> "...a risk of improper synthesis."

Reply:
This is longstanding text and I'm inclined to agree with the current wording - "risk of improper synthesis" seems more opaque. Maybe "a risk of the design not getting synthesized properly" to improve the grammar